app/models/gsa18f/proposal_form.rb

```diff
@@ -0,0 +1,124 @@
+#module Ncr
+module Gsa18f
+  DATA = YAML.load_file("#{Rails.root}/config/data/18f.yaml")
+
+  class ProposalForm
+    include SimpleFormObject
+
+    EXPENSE_TYPES = %w(BA61 BA80)
+
+    URGENCY = DATA['URGENCY']
+    OFFICES = DATA['OFFICES']
+    attribute :origin, :string
+    attribute :amount, :decimal
+    attribute :approver_email, :text
+    attribute :description, :text
+    attribute :expense_type, :text
+    attribute :requester, :user
+    attribute :vendor, :string
+    attribute :not_to_exceed, :boolean
+    attribute :building_number, :string
+    attribute :rwa_number, :string
+    
+    
+    #18f additions
+    attribute :office, :string
+    attribute :justification, :text
+    attribute :link_to_product, :string
+    attribute :quantity, :decimal
+    attribute :date_requested, :string
+    attribute :urgency, :string
+    attribute :additional_info, :string
+    attribute :cost_per_unit, :decimal
+    attribute :product_name_and_description, :text
+
+    validates :cost_per_unit, numericality: {
+      greater_than_or_equal_to: 0,
+      less_than_or_equal_to: 3000
+    }
+    validates :approver_email, presence: true
+    validates :product_name_and_description, presence: true
+    #validates :expense_type, inclusion: {in: EXPENSE_TYPES}, presence: true
+    #validates :requester, presence: true
+    #validates :vendor, presence: true
+    #validates :building_number, presence: true
+    #validates :office, presence: true
+
+    def budget_approver_email
+      ENV['NCR_BUDGET_APPROVER_EMAIL'] || 'communicart.budget.approver@gmail.com'
+    end
+
+    def finance_approver_email
+      ENV['NCR_FINANCE_APPROVER_EMAIL'] || 'communicart.ofm.approver@gmail.com'
+    end
+
+    def requires_finance_approval?
+      self.expense_type == 'BA61'
+    end
+
+    def approver_emails
```

Comment:
In the 18F case, I'm thinking we can just make Ric the approver_email and not even require the requester to type in his email address. This means that 1) removal from the form, and 2) updating `approver_emails` to be `['rics@email_address']`